src/lib.rs

```diff
@@ -15,6 +15,10 @@
 #![no_std]
 #![deny(missing_docs)]
 
+#[cfg(test)]
+#[macro_use]
+extern crate alloc;
```

Comment:
Why do you need this?

Reply:
Oh, I think I don't. I must have put it there when testing something (maybe using `vec!` somewhere) and it should be safe to remove.